charts/kargo/values.yaml

```diff
@@ -268,6 +268,16 @@ controller:
     ## @param controller.globalCredentials.namespaces List of namespaces to look for shared credentials.
     namespaces: []
 
+  committer:
+    ## @param controller.committer.name Specifies the name of the Kargo controller committer.
+    name: "Kargo Render"
+    ## @param controller.committer.email Specifies the email of the Kargo controller committer.
+    email: "kargo-render@akuity.io"
+
+    signingKeySecret:
+    ## @param controller.committer.signingKeySecret.name Specifies the name of an existing `Secret` which contains the committer's signing key. The value should be accessible under `.data.signingKey` in the same namespace as Kargo.
+      name: ""
```

Comment:
We can probably do without nesting `signingKeySecret.name`. Can `signingKeySecret` just be a string field instead of an object? _Or_ were you anticipating that there may be other settings that get added here in the future?

Reply:
sgtm

Reply:
Already have an example 😄 

[feat(charts): add signing key type to indicate how to sign commits wi…](https://github.com/akuity/kargo/pull/1843/commits/08134a4cba6301de5dbe24b30e7617ff65cf2cc1)